code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1192,7 +1192,9 @@
 		var/list/valid_facial_hair = valid_sprite_accessories(facial_hair_styles_list, null, species.name)
 		if (!(my_appearance.f_style in valid_facial_hair))
 			my_appearance.f_style = random_facial_hair_style(gender, species)
-		if (my_appearance.s_tone > species.max_skin_tone)
+		if ((my_appearance.s_tone < 0) && !(species.anatomy_flags & HAS_SKIN_TONE))
+			my_appearance.s_tone = random_skin_tone(species)
+		else if (my_appearance.s_tone > species.max_skin_tone)
```

Comment:
why is this not just a `||` clause in the previous if statement? The code inside is exactly the same.

Reply:
small brain moment